projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/protect-value-section.tsx

```diff
@@ -0,0 +1,113 @@
+
+import './style.scss';
+
+const ProtectValueSection: React.FunctionComponent = () => {
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { status } = detail;
+	const isProtectActive =
+		status === PRODUCT_STATUSES.ACTIVE || status === PRODUCT_STATUSES.CAN_UPGRADE;
+
+	return isProtectActive ? <WithProtectValueSection /> : <NoProtectValueSection />;
+};
+
+export default ProtectValueSection;
+
+const WithProtectValueSection: React.FunctionComponent = () => {
+	const { protectStatus } = getMyJetpackWindowInitialState();
+	const lastScanTime = protectStatus?.last_checked;
+	const timeSinceLastScan = lastScanTime ? timeSince( Date.parse( lastScanTime ) ) : '...';
+
+	const lastScanText = sprintf(
+		/* translators: %s is how long ago since the last scan took place, i.e.- "17 hours ago" */
+		__( 'Last scan: %s', 'jetpack-my-jetpack' ),
+		timeSinceLastScan
+	);
+	return <ValueSection isProtectActive={ true } lastScanText={ lastScanText } />;
+};
+
+const NoProtectValueSection: React.FunctionComponent = () => {
+	const { plugins, themes } = getMyJetpackWindowInitialState();
+	const pluginsCount = Object.keys( plugins ).length;
+	const themesCount = Object.keys( themes ).length;
+
+	const pluginsThemesText = sprintf(
+		/* translators: %1$d is the number (integer) of plugins and %2$d is the number (integer) of themes the site has. */
+		__( '%1$s plugins & %2$s themes', 'jetpack-my-jetpack' ),
+		pluginsCount,
+		themesCount
+	);
+
+	return <ValueSection isProtectActive={ false } lastScanText={ pluginsThemesText } />;
+};
+
+const ValueSection: React.FunctionComponent< {
+	isProtectActive: boolean;
+	lastScanText: string;
+} > = ( { isProtectActive, lastScanText } ) => {
+	const isMobileViewport: boolean = useViewportMatch( 'medium', '<' );
+	const [ isVisiblePopover_PluginsThemes, setIsVisiblePopover_PluginsThemes ] = useState( false );
+
+	const togglePopover_PluginsThemes = function () {
+		setIsVisiblePopover_PluginsThemes( state => ! state );
+	};
+
+	return (
+		<>
+			<div className="value-section__last-scan">
+				<span>{ lastScanText }</span>
+				{ ! isProtectActive && (
+					<span>
+						<button
+							className="value-section__tooltip-button"
+							// eslint-disable-next-line react/jsx-no-bind
+							onClick={ togglePopover_PluginsThemes }
+						>
+							<Gridicon icon="info-outline" size={ 14 } />
+							{ isVisiblePopover_PluginsThemes && (
+								<Popover
+									placement={ isMobileViewport ? 'top-end' : 'right' }
+									noArrow={ false }
+									offset={ 10 }
+								>
+									<p className="value-section__tooltip-heading">
+										{ __( 'Improve site safety: secure plugins & themes', 'jetpack-my-jetpack' ) }
+									</p>
+									<p className="value-section__tooltip-content">
+										{ __(
+											'Your site has 14 plugins and 3 themes lacking security measures. Improve your site’s safety by adding protection at no cost.',
```

Comment:
I see the 14 and 3 are hardcoded here. Are these going to be changed later? I don't really see a need to postpone it when we are dynamically showing them on the card itself

Reply:
Yes, I've refactored some of this PR to show the dynamic data within the tooltips. 👍